src/main/resources/META-INF/plugin.xml

```diff
@@ -104,6 +104,8 @@
 
         <languageSupport language="JSON" implementationClass="tanvd.grazi.ide.language.json.JsonSupport"/>
 
+        <languageSupport language="Properties" implementationClass="tanvd.grazi.ide.language.props.PropsSupport"/>
```

Comment:
It looks like `Properties` plugin can be disabled. So we better move it to optional. (also move there JSON and XML)

Reply:
Done